test/parallel/test-repl-tab-complete.js

```diff
@@ -305,6 +306,34 @@ testMe.complete('.b', common.mustCall((error, data) => {
   assert.deepStrictEqual(data, [['break'], 'b']);
 }));
 
+// tab completion for large buffer
+[ Array, Buffer ].forEach((type) => {
+  putIn.run(['.clear']);
+
+  if (type === Array) {
+    putIn.run(['var ele = []; for (let i = 0; i < 1e7; i++) ele.push(i);']);
+  } else {
+    putIn.run(['var ele = Buffer.alloc(1e8)']);
+  }
+
+  common.hijackStderr(common.mustCall((err) => {
+    process.nextTick(function() {
```

Comment:
Small and not essential nit: could you please use an arrow function here to be consistent with the surrounding code?